Libraries/Lists/FlatList.js

```diff
@@ -125,14 +125,39 @@ type OptionalProps<ItemT> = {|
   /**
    * Multiple columns can only be rendered with `horizontal={false}` and will zig-zag like a
    * `flexWrap` layout. Items should all be the same height - masonry layouts are not supported.
+   *
+   * The default value is 1.
+   */
+  numColumns?: number,
+  /**
+   * Note: may have bugs (missing content) in some circumstances - use at your own risk.
+   *
+   * This may improve scroll performance for large lists.
+   *
+   * The default value is true for Android.
    */
-  numColumns: number,
+  removeClippedSubviews?: boolean,
   /**
    * See `ScrollView` for flow type and further documentation.
    */
   fadingEdgeLength?: ?number,
 |};
 
+/**
+ * Default Props Helper Functions
+ * Use the following helper functions for default values
+ */
+
+// removeClippedSubviewsOrDefault(this.props.removeClippedSubviews)
+function removeClippedSubviewsOrDefault(removeClippedSubviews: ?boolean) {
+  return removeClippedSubviews || Platform.OS === 'android';
```

Comment:
I think this needs to be `removeClippedSubviews == null ? Platform.OS === 'android' : removeClippedSubviews`
Or similarly `removeClippedSubviews ?? Platform.OS === 'android'`